packages/main/src/SelectPopover.hbs

```diff
@@ -70,8 +68,9 @@
 		hide-arrow
 		class="ui5-valuestatemessage-popover"
 		placement-type="Bottom"
+		horizontal-align="Left"
 	>
-		<div slot="header" class="ui5-responsive-popover-header {{classes.popoverValueState}}" style="{{styles.popoverHeader}}">
```

Comment:
ui5-responsive-popover-header class should remain. It is used in ```ResponsivePopoverCommonCss```

Reply:
We are not using its default styles. If I keep it I have to add overrides for everything. We use different width, padding, font-size.